airflow/providers/databricks/operators/databricks.py

```diff
@@ -1118,14 +1012,16 @@ def _get_run_json(self) -> dict[str, Any]:
             raise ValueError("Must specify either existing_cluster_id or new_cluster.")
         return run_json
 
-    def launch_notebook_job(self) -> int:
+    def launch_job(self) -> int:
```

Comment:
I personally prefer to keep method prefixes with _ when there is little chance that users will use them directly. This makes it easier to maintain and avoids concerns about future deprecation.

Reply:
done so, please check.